tools/accounts-balance-checker/pkg/esclient/bulk.go

```diff
@@ -0,0 +1,106 @@
+package esclient
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/elastic/go-elasticsearch/v7/esapi"
+)
+
+var log = logger.GetOrCreate("client")
+
+// DoBulkRequest will do a bulk of request to elastic server
+func (ec *esClient) DoBulkRequest(buff *bytes.Buffer, index string) error {
+	reader := bytes.NewReader(buff.Bytes())
+
+	options := make([]func(*esapi.BulkRequest), 0)
+	if index != "" {
+		options = append(options, ec.client.Bulk.WithIndex(index))
+	}
+
+	res, err := ec.client.Bulk(
+		reader,
+		options...,
+	)
+	if err != nil {
+		return err
+	}
+	if res.IsError() {
+		return fmt.Errorf("%s", res.String())
+	}
+
+	defer closeBody(res)
+
+	bodyBytes, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return err
+	}
+	return extractErrorFromResponseBytes(bodyBytes)
+}
+
+// bulkRequestResponse defines the structure of a bulk request response
+type bulkRequestResponse struct {
+	Errors bool `json:"errors"`
+	Items  []struct {
+		ItemIndex  *Item `json:"index"`
+		ItemUpdate *Item `json:"update"`
+	} `json:"items"`
+}
+
+type Item struct {
```

Comment:
missing comment.. or make it unexported, seems to be used only here

Reply:
done